apps/studio/components/interfaces/Settings/Logs/LogSelection.tsx

```diff
@@ -75,6 +77,10 @@ const LogSelection = ({ log, onClose, queryType, isLoading, error }: LogSelectio
           ...log,
         }
         return <DefaultPreviewSelectionRenderer log={postgresLog} />
+      case 'pg_upgrade':
+        if (!log) return null
+        if (!log.metadata) return <DefaultPreviewSelectionRenderer log={log} />
+        return <PgUpgradeSelectionRenderer log={log} />
```

Comment:
this is the only part which im not sure what to expect from the UI POV without some data to test on the preview